plugins/device-auth-backend/config.d.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  deviceAuth?: {
+    /**
+     * The interval in seconds at which to instruct the client
+     * device to poll for tokens
+     */
+    initialInterval: number;
```

Comment:
feel free to use `HumanDuration | string` like [here](https://github.com/backstage/backstage/blob/1ac6c623d0719c0822094e72e431803356fdf016/packages/backend-plugin-api/src/services/definitions/SchedulerService.ts#L209), and parse with the standard utilities like [here](https://github.com/backstage/backstage/blob/1ac6c623d0719c0822094e72e431803356fdf016/packages/backend-plugin-api/src/services/definitions/SchedulerService.ts#L386). Then you can convert those to milliseconds of you want, like [here](https://github.com/backstage/backstage/blob/1ac6c623d0719c0822094e72e431803356fdf016/plugins/catalog/src/apis/EntityPresentationApi/DefaultEntityPresentationApi.ts#L213).